plugins/modules/ec2_win_password.py

```diff
@@ -181,7 +181,7 @@ def main():
             module.fail_json(msg="unable to parse key file")
     elif key_data is not None and key_file is None:
         try:
-            key = load_pem_private_key(key_data, b_key_passphrase, default_backend())
+            key = load_pem_private_key(key_data.encode('ascii'), b_key_passphrase, default_backend())
```

Comment:
It looks like `load_pem_private_key` expects a bytes like object so the key_data can just use to_bytes (like b_key_passphrase).

Reply:
This is how I did it for b_key_passphrase:

```
b_key_passphrase = to_bytes(module.params.get('key_passphrase'), err    ors='surrogate_or_strict')
```

OMG - I haven't seen this code since 2016.